src/Symfony/Component/Form/FormFactory.php

```diff
@@ -61,36 +61,12 @@ public function createForProperty($class, $property, $data = null, array $option
      */
     public function createBuilder($type = 'Symfony\Component\Form\Extension\Core\Type\FormType', $data = null, array $options = array())
     {
-        $name = null;
-        $typeName = null;
-
         if ($type instanceof ResolvedFormTypeInterface) {
```

Comment:
I think there is a bug in the current implementation. createBuilder has logic to handle ResolvedFormTypeInterface, but it then calls createNamedBuilder which will throw an exception as it requires a string as the type.